libevmcore/Instruction.cpp

```diff
@@ -204,6 +54,8 @@ static const std::map<Instruction,  InstructionInfo> c_instructionInfo =
 	{ Instruction::ADDMOD,       { "ADDMOD",         0,     3,    1,  false,       Tier::Mid } },
 	{ Instruction::MULMOD,       { "MULMOD",         0,     3,    1,  false,       Tier::Mid } },
 	{ Instruction::SIGNEXTEND,   { "SIGNEXTEND",     0,     2,    1,  false,       Tier::Low } },
+	{ Instruction::RETURNDATASIZE,{"RETURNDATASIZE", 0,     0,    1,  false,       Tier::VeryLow } },
```

Comment:
Are the prices still not right? `Tier::VeryLow` is 3, but `RETURNDATASIZE`  should be 2, which is `Tier::Base`, like `CALLDATASIZE`
And `RETURNDATACOPY` should be `Tier::VeryLow` ?